packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -174,39 +173,43 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-.user-menu-list li {
-  align-items: center;
-  display: flex;
-  margin: var(--oc-space-xsmall) 0;
+.user-menu-list {
+  padding: 10px;
```

Comment:
Please use `--oc-space-...` vars whenever possible. If you need exactly 10 pixels you can use `calc` to add 2px to an existing var... but would prefer to stay in the `8px` sizing base.